Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM rust:1.61 as builder
+
+WORKDIR /app
+COPY . /app
+RUN cargo build --release --all-features --bin bindle-server
+
+FROM gcr.io/distroless/cc
+
+VOLUME [ "/bindle-data" ]
+
+ENV BINDLE_IP_ADDRESS_PORT="0.0.0.0:8080"
+ENV BINDLE_DIRECTORY="/bindle-data/bindles"
+
+COPY --from=builder /app/target/release/bindle-server /
```

Comment:
One small nit: We might want to create and run as a `bindle` user instead of `root` as that is generally good for hygiene. But that is not blocking

Reply:
Updated it. Thank you.